.changelog/679a53c14cfe4943867d8e23cc85abb9.json

```diff
@@ -0,0 +1,8 @@
+{
+    "id": "679a53c1-4cfe-4943-867d-8e23cc85abb9",
+    "type": "feature",
+    "description": "`AddSDKAgentKey` and `AddSDKAgentKeyValue` in `aws/middleware` package have been updated to direct metadata to `User-Agent` HTTP header.",
+    "modules": [
+        "."
```

Comment:
The reason for not doing this is mainly to avoid having to deal with that condition in multiple spots, "." automatically plays nice with `path` and `filepath` as you would expect when you are joining this relative locations to the repository path on the filesystem.